src/electron/views/tab-stops/tab-stops-testing-content.tsx

```diff
@@ -30,31 +30,27 @@ export const TabStopsTestingContent = NamedFC<TabStopsTestingContentProps>(
                     <strong>How to test:</strong>
                 </span>
                 <ol className={styles.howToTestList}>
-                    <li>Navigate to the screen you want to test in your app.</li>
+                    <li>In your target app, navigate to the screen you want to test.</li>
                     <li>
-                        Turn on the Show tab stops toggle. An empty circle will highlight the
-                        element with focus.
+                        Turn on the <strong>Show tab stops</strong> toggle. An empty circle will
+                        highlight the element in the target app that currently has input focus.
                     </li>
                     <li>
-                        If you are connected to an emulator, select your emulator app to bring it
-                        into focus and use your keyboard.
+                        If you're testing on a virtual device, switch to the emulator app by
+                        clicking it or pressing Alt+Tab.
```

Comment:
On a Mac, this would be Command+Tab. I'm not sure how Windows-centric we want to be

Reply:
"If you're testing on a virtual device, switch to the emulator app by clicking it or by pressing **Alt+Tab** (Windows or Linux) or **Command+Tab** (Mac)."

This is the approved text from Lisa Louise.